Wikipedia/UI-V5/WMFTableOfContentsViewController.swift

```diff
@@ -95,19 +95,23 @@ public class WMFTableOfContentsViewController: UITableViewController, WMFTableOf
         }
     }
 
+    // MARK: - Header
+    func forceUpdateHeaderFrame(){
+        //See reason for fix here: http://stackoverflow.com/questions/16471846/is-it-possible-to-use-autolayout-with-uitableviews-tableheaderview
+        self.tableView.tableHeaderView!.setNeedsLayout()
+        self.tableView.tableHeaderView!.layoutIfNeeded()
+        let headerHeight = self.tableView.tableHeaderView!.systemLayoutSizeFittingSize(UILayoutFittingCompressedSize).height
+        var headerFrame = self.tableView.tableHeaderView!.frame;
+        headerFrame.size.height = headerHeight
+        self.tableView.tableHeaderView!.frame = headerFrame;
+        self.tableView.tableHeaderView = self.tableView.tableHeaderView
```

Comment:
I read the SO post, and see what you're doing, but still...
![wat](http://i.imgur.com/wm9Srgj.gif)